src/app/shell/personal-cabinet/shared-cabinet/applications/applications.component.ts

```diff
@@ -159,4 +140,9 @@ export class ApplicationsComponent implements OnInit, OnDestroy, AfterViewInit {
     this.applicationParams.statuses = statuses;
     this.applicationParams.showBlocked = tabIndex === ApplicationStatusTabParams.Blocked;
   }
+
+  private onGetApplications(): void {
```

Comment:
wrong na,ing. This is not a method that is called on some action 
onClick
onSubmit
onCancel

this is private method that does inner logic, so it should be named as getApplication()

Reply:
onSomething - this is an approach for naming methods that are called from the template